src/new/wallet.js

```diff
@@ -850,16 +847,10 @@ class HathorWallet extends EventEmitter {
    * @yields {Utxo} all available utxos
    */
   async * getAllUtxos(options = {}) {
```

Comment:
Where do we use this method? Would be much better to call it `getAvailableUtxos`

Reply:
Renamed to `getAvailableUtxos`